src/app/modules/time-entries/components/calendar/calendar.component.ts

```diff
@@ -34,97 +46,115 @@ export class CalendarComponent implements OnInit {
   timeEntriesAsEvent: CalendarEvent[];
   nextDateDisabled: boolean;
 
-  constructor() {
+  constructor(private referenceChangeDetector: ChangeDetectorRef) {
     this.initialDate = new Date();
     this.previusDate = new Date();
     this.isToday = false;
     this.timeEntriesAsEvent = [];
     this.nextDateDisabled = true;
-   }
+  }
 
   ngOnInit(): void {
     this.isToday = this.isVisibleForCurrentDate();
     this.navigationEnable(this.calendarView);
   }
 
-  get CalendarViewEnum(): typeof CalendarView{
+  get CalendarViewEnum(): typeof CalendarView {
     return CalendarView;
   }
 
+  scrollToCurrentTimeMarker() {
+    console.log('Si entro');
+    if (this.calendarView === CalendarView.Week || CalendarView.Day) {
+      console.log(`Calendar View ${this.calendarView}`);
```

Comment:
Suggestions - do not add any console log

Reply:
I forgot, sorry 😭

Reply:
I have already corrected the comments you gave me.